client/pruning/main.go

```diff
@@ -21,41 +21,51 @@ const FlagAppDBBackend = "app-db-backend"
 
 // Cmd prunes the sdk root multi store history versions based on the pruning options
 // specified by command flags.
-func Cmd(appCreator servertypes.AppCreator) *cobra.Command {
+func Cmd(appCreator servertypes.AppCreator, defaultNodeHome string) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "prune",
+		Use:   "prune [pruning-method]",
 		Short: "Prune app history states by keeping the recent heights and deleting old heights",
 		Long: `Prune app history states by keeping the recent heights and deleting old heights.
```

Comment:
Can we confirm the output of this displays nicely and is formatted correctly?

Reply:
There is a spacing issue, let me fix it.